### PR TITLE
[benchmark] add cpu utilization

### PR DESCRIPTION
**Pull request by chungjin**
Append cpu benchmark test result to the site page.

eg:
Before the change: 
https://minikube.sigs.k8s.io/docs/benchmarks/timetok8s/v1.25.1/

After the change:
append following markdown table and chart to the page:
image

### CODE REVIEW

**Comment by klaases** [comment]
Hi @medyagh, are you the right person to verify this patch?

**Review by spowelljr — CHANGES_REQUESTED** [review]
You can prevent the graph from going into the legend by tracking the maximum value and then appending a value to it to set the height. For example the highest value + 3 would probably give proper clearance of the legend in your case.

Here's how I'm doing this for reference: https://github.com/kubernetes/minikube/blob/master/hack/benchmark/time-to-k8s/public-chart/generate-chart.go#L280

**Comment by chungjin** [comment]
> You can prevent the graph from going into the legend by tracking the maximum value and then appending a value to it to set the height. For example the highest value + 3 would probably give proper clearance of the legend in your case.
> 
> Here's how I'm doing this for reference: https://github.com/kubernetes/minikube/blob/master/hack/benchmark/time-to-k8s/public-chart/generate-chart.go#L280

Thank you @spowelljr for the suggestion! I have applied it and update the graph.

File: hack/benchmark/time-to-k8s/chart.go [hunk 1]
```diff
@@ -92,14 +75,15 @@ func readInCSV(csvPath string, apps map[string]runs) error {
 		values := []float64{}
 
 		// 8-13 contain the run results
```
Comment (spowelljr) [inline hack/benchmark/time-to-k8s/chart.go:77]: Just update this comment to match the range we're using.

File: hack/benchmark/time-to-k8s/cpu.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+	p := plot.New()
+	p.Title.Text = "CPU utilization to go from 0 to successful Kubernetes deployment"
+	p.Y.Label.Text = "CPU utilization"
+	setYMax(p, values)
+
+	w := vg.Points(20)
+
+	barsA, err := plotter.NewBarChart(values[0], w)
+	if err != nil {
+		panic(err)
```
Comment (spowelljr) [inline hack/benchmark/time-to-k8s/cpu.go:66]: Any reason we're panicking instead of just returning an error?

**Review by spowelljr — CHANGES_REQUESTED** [review]

File: hack/benchmark/time-to-k8s/cpu.go (hunk 2 above)
Comment (medyagh) [reply hack/benchmark/time-to-k8s/cpu.go:66]: I agree lets return error instead of panic here

**Review by medyagh — CHANGES_REQUESTED** [review]

File: hack/benchmark/time-to-k8s/chart.go (hunk 1 above)
Comment (chungjin) [reply hack/benchmark/time-to-k8s/chart.go]: addressed

File: hack/benchmark/time-to-k8s/cpu.go (hunk 2 above)
Comment (chungjin) [reply hack/benchmark/time-to-k8s/cpu.go]: done

**Review by spowelljr — APPROVED** [review]

**Pull request merged**